libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-bar-filter-modal-config.ts

```diff
@@ -1,9 +1,22 @@
 import { Type } from '@angular/core';
 
+import { SkyFilterBarFilterModalSizeType } from './filter-bar-filter-modal-size';
+
 export interface SkyFilterBarFilterModalConfig {
+  /**
+   * The modal component to display when the user selects the filter.
+   * The component should expect a `SkyFilterBarFilterModalContext` object to be provided by the modal service on instantiation.
```

Comment:
```suggestion
   * The component expects the modal service to provide a `SkyFilterBarFilterModalContext` object on instantiation.
```
The "should" phrasing makes it sound like this is optional. I don't think it is, so I reworded to avoid that implication. If I'm wrong and it is optional, we should ignore my suggestions but should also expand the description to explain what the other options are or change the wording to make this a recommendation/best practice.